packages/itwinui-css/backstop/tests/tabs.html

```diff
@@ -68,7 +68,8 @@ <h2>Default</h2>
                 aria-hidden="true"
               ></svg-placeholder>
               <span class="iui-tab-label">
-                <div>Active</div>
+                <div>This is a very long 'Active' tab content, which hopefully will wrap if all tab headers content is
```

Comment:
this is really messing up the demos. should we add new ones instead of changing the existing ones?

also the text probably doesn't need to be as long

Reply:
I would also prefer if the first demo looks normal, maybe we can use the third demo of each category for long labels?

Reply:
I would also prefer if the first demo looks normal, maybe we can use the third demo of each category for long labels?

Reply:
Added separate demos with long text